packages/cli/src/ui/components/messages/ToolGroupMessage.test.tsx

```diff
@@ -81,14 +84,21 @@ describe('<ToolGroupMessage />', () => {
   const baseProps = {
     groupId: 1,
     terminalWidth: 80,
-    config: mockConfig,
     isFocused: true,
   };
 
+  // Helper to wrap component with required providers
+  const renderWithProviders = (component: React.ReactElement) =>
```

Comment:
is this needed? if so can you switch to the standard wrap with providers from the test helpers? suspect this isn't needed as this change was briefly in main but backed out as I recall.